tools/cli/commands.go

```diff
@@ -1155,7 +1149,7 @@ func ObserveHistory(c *cli.Context) {
 // ObserveHistoryWithID show the process of running workflow
 func ObserveHistoryWithID(c *cli.Context) {
 	if !c.Args().Present() {
-		ExitIfError(errors.New("workflow_id is required"))
+		ErrorAndExit("workflow_id is required.", nil)
```

Comment:
Would be better to start all Error msg with Upper case, or add "Argument workflow_id ..."

Reply:
Updated. Will send out a new PR about the change.